### PR TITLE
Add volume claim for StatefulSet

### PR DESCRIPTION
**Pull request by hanahmily**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,13 @@ Changes by Version
 ==================
 Release Notes.
 
+0.2.0
+------------------
+
+#### Bugs
+
+- Add volume claim for StatefulSet
```
Comment (wu-sheng) [inline CHANGES.md:10]: We have other changes for 0.2, right?

Comment (hanahmily) [reply CHANGES.md:10]: No, we don't have others. Do you tend to move them to 0.1.1 ?

Comment (wu-sheng) [reply CHANGES.md:10]: Such as feature like adding support to cluster mode?

Comment (wu-sheng) [reply CHANGES.md:10]: I mean these commits, https://github.com/apache/skywalking-banyandb-helm/compare/v0.1.0...master

Comment (hanahmily) [reply CHANGES.md]: done

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**